install.sh

```diff
@@ -29,14 +29,10 @@ if [ "$(id -u)" != "0" ]; then
     [[ "$0" = "$BASH_SOURCE" ]] && exit 1 || return 1
 fi
 
-# Split path to this install script into an array.
-IFS='/' SRC_PATH=($0)
-# Remove this file name.
-unset SRC_PATH[-1]
-# Grab the directory name.
-APP_DIR="${SRC_PATH[-1]}"
-# Joint path back together.
-SRC_PATH="${SRC_PATH[*]}"
+# source path to find current directory and to copy application to /opt/
+SRC_PATH=$(pwd)
```

Comment:
Will this require cd to dir with install.sh? I'd like to be able to run the install from elsewhere. like:
sudo stuff/ci_edit/install.sh
Maybe cd within the script itself (similar to what PR from crabime does)?

Reply:
This did require the `install.sh` to be run in the package directory.  However, I just amended the latest commit, and now you can execute the install script from any directory, and the update will successfully install to `/opt/` using a subdirectory as the package directory.